pkg/providers/ingress/translation/translator.go

```diff
@@ -356,7 +356,7 @@ func (t *translator) translateDefaultUpstreamFromIngressV1(namespace string, bac
 	ups.ID = id.GenID(ups.Name)
 	return ups
 }
-func (t *translator) translateUpstreamFromIngressV1(namespace string, backend *networkingv1.IngressServiceBackend) (*apisixv1.Upstream, error) {
+func (t *translator) translateUpstreamFromIngressV1(namespace string, backend *networkingv1.IngressServiceBackend, eventType types.EventType) (*apisixv1.Upstream, error) {
 	var svcPort int32
 	if backend.Port.Name != "" {
```

Comment:
If `backend.port.name` is used here, it will cause repeated attempts, including use cases where endpoints do not exist. 
I recommend implementing this method instead of passing parameters to cause logic coupling
https://github.com/apache/apisix-ingress-controller/commit/486b46abd1c9665005e83036433af213aaa28f21

Reply:
done, reuse the old translate